bin/oref0-ed-battery.sh

```diff
@@ -2,7 +2,6 @@
 
 echo Run oref0-ed-battery
 if ! /home/edison/src/EdisonVoltage/voltage json batteryVoltage battery | grep "-66" 2>/dev/null; then
-    echo start Voltage server and trigger
-    cd /home/edison/src/EdisonVoltage && sudo ./voltage_server &
-    cd /home/edison/src/EdisonVoltage && echo 1 > /tmp/battery_trigger
+    echo start Voltage
+    sudo /home/edison/src/EdisonVoltage/voltage
```

Comment:
Why are you installing this in crontab?  It can be added as a device 

[device "edison-battery"]
fields = output
cmd = sudo /root/loopy/EdisonVoltage/voltage
vendor = openaps.vendors.process
args =

Then just refresh the report inside of the monitor-pump alias